python/py-pytest-metadata/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       pytest-metadata
+name                py-${real_name}
+version             1.8.0
+revision            0
+categories-append   devel
+
+license             Mozilla-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Plugin for accessing test session metadata
+
+long_description    pytest-metadata is a plugin for pytest that\
+                    provides access to test session metadata.
+
+homepage            https://pypi.python.org/pypi/${real_name}
+
+master_sites        pypi:p/${real_name}/
+distname            ${real_name}-${version}
+
+checksums           rmd160  722a1d9d4121fd5314e3eeda69e678198af06ae6 \
+                    sha256  2071a59285de40d7541fde1eb9f1ddea1c9db165882df82781367471238b66ba \
+                    size    9227
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools_scm
+
+    depends_run-append \
+                    port:py${python.version}-pytest
+
+    depends_test-append \
+                    port:py${python.version}-pytest
```

Comment:
this is ```depends_test``` is redundant here since ```py-pytest``` is already listed as ```depends_run```

Reply:
Thanks done.